src/osd/PrimaryLogPG.cc

```diff
@@ -13970,7 +13970,7 @@ uint64_t PrimaryLogPG::recover_backfill(
 	MOSDPGScan *m = new MOSDPGScan(
 	  MOSDPGScan::OP_SCAN_GET_DIGEST, pg_whoami, e, get_last_peering_reset(),
 	  spg_t(info.pgid.pgid, bt.shard),
-	  pbi.end, hobject_t());
+	  pbi.end, hobject_t(), eversion_t());
```

Comment:
This constructor constructs the request, right?  Can we just not include the version field?  Might make sense to have two different named constructors, one for GET_DIGEST, the other for GET_DIGEST_REPLY?